app/3.0/docs/stack-blitz.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Polymer 3.0 on StackBlitz"
+---
+
+<!-- toc -->
+
+[StackBlitz](https://stackblitz.com) is an online editor for web projects. You can use StackBlitz to prototype and preview Polymer elements.
+
```

Comment:
Maybe we should add a quick summary of what StackBlitz does that other online editors don't?

In particular, it transpiles ES6 to ES5 so the samples can run on any browser we support; and that it lets you install dependencies using npm and supports node-style module resolution, so you can preview code without a build step.

You can use Polymer 3 in other online editors (like Plunker or jsbin), but the code may not be transpiled for older browsers, and you may need to use non-standard paths to import dependencies. 

I might not have said that very well. But, um. I think that's why we chose it instead of using other things.